data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java

```diff
@@ -61,7 +61,7 @@ public void setUp() {
         collectorMock = mock(MessageCollector.class);
         contextMock = mock(TaskContext.class);
         metricsRegistry = Mockito.mock(MetricsRegistry.class);
-        counter = Mockito.mock(Counter.class);
+        counter = new Counter("Counter");
```

Comment:
The counter need not be a concrete object.

Reply:
Made changes for counter